### ISSUE TITLE
![BUG]! Error 500 when opening My Timesheets

### ISSUE THREAD

**Issue by jiriks74** [Leantime/leantime#1914]
**What set up are you using**
* [x] Self Hosted

**Describe the bug**
When opening `My Timesheets` I get `HTTP 500 internal server error`
![image](https://github.com/Leantime/leantime/assets/54378412/f67dbc42-9a18-4075-b3eb-aa5fea4d32ba)

**To Reproduce**
1. Login
2. Click on `My Timesheets`
3. See error

**Expected behavior**
`My Timesheets` site being successfully loaded

**Leantime Version**
v2.4-beta-4

**Server**
Docker

**PHP / MySQL Version**
Docker

**Comment by jiriks74** [comment]
After adding `use DateTime;` to the file I get a different error related to user handling (I think).

I'll provide info when I'll have time to use a PC.

**Comment by jiriks74** [comment]
The new error:
```php
Target class [Leantime\Domain\Tickets\Repositories\Users] does not exist.

in[ /var/www/html/app/Domain/Tickets/Repositories/Tickets.php ](https://leantime.stefka.eu/timesheets/showMy)(line 280)

                return $values;
            }
            public function getUsersTickets($id, $limit)
            {
                $users = app()->make(Users::class);
                $user = $users->getUser($id);
                $sql = "SELECT
                            ticket.id,
                            ticket.headline,
```

![image](https://github.com/Leantime/leantime/assets/54378412/ed60092a-18e9-4b45-abf3-5020f1b9e4ab)

**Comment by jiriks74** [comment]
Fixed in #1915
